.github/ISSUE_TEMPLATE/000_icon_request.yml

```diff
@@ -1,36 +1,60 @@
 name: Icon request
-description: Suggest an icon to be included in Font Awesome
+description: Suggest an icon, modifier, or emoji to be included in Font Awesome
 labels: ["new icon"]
-title: "Icon request: "
+projects: ["FortAwesome/20"]
+title: "Icon Request: "
 body:
   - type: markdown
     attributes:
-      value: |
-        Thanks for taking the time to request a new icon!
-  - type: textarea
-    id: use-case
-    attributes:
-      label: Use case
-      description: Describe how this icon could be used.
-  - type: textarea
-    id: example
+      value: "## Before You Request"
+  - type: markdown
     attributes:
-      label: Example image
-      description: Insert the example image here if necessary. A good example is a single color image which plays well at small sizes (16px)
+      value: |
+      Thanks for requesting a new Font Awesome icon! But first, let's get some specifics out of the way. For example, lots of amazing custom icons can already be created using our [Icon Wizard](https://blog.fontawesome.com/icon-wizard/)! Please check through this list first to ensure your request is as useful and unique as possible:
```

Comment:
I think this might be our issue, even though it’s nearly identical to how the old version was handled. My guess is line 13 needs to be indented to nest under the Value key